weave/__init__.py

```diff
@@ -24,19 +24,15 @@
 from .index.index_sqlite import IndexSQLite
 from .index.index_sql import IndexSQL
 from .pantry import Pantry
-from .pantry_factory import create_pantry
 from .mongo_loader import MongoLoader
 
-
-__version__ = "1.9.1"
-
+__version__ = "1.9.2"
 
 __all__ = [
     "Basket",
     "IndexPandas",
     "IndexSQLite",
     "IndexSQL",
     "Pantry",
-    "MongoLoader",
-    "create_pantry",
```

Comment:
Is there a reason we remove create_pantry?

Reply:
I have no idea, I didn't make any changes

Reply:
Must've been when you merged main into your branch. Please re-add the lines for create_pantry and the import above